src/golang/cmd/server/handler/list_operators_for_integration.go

```diff
@@ -57,18 +66,51 @@ func (*ListOperatorsForIntegrationHandler) Name() string {
 	return "ListOperatorsForIntegration"
 }
 
-func (*ListOperatorsForIntegrationHandler) Prepare(r *http.Request) (interface{}, int, error) {
+func (h *ListOperatorsForIntegrationHandler) Prepare(r *http.Request) (interface{}, int, error) {
+	aqContext, statuscode, err := aq_context.ParseAqContext(r.Context())
+	if err != nil {
+		return nil, statuscode, err
+	}
+
 	integrationIDStr := chi.URLParam(r, routes.IntegrationIdUrlParam)
 	integrationID, err := uuid.Parse(integrationIDStr)
 	if err != nil {
 		return nil, http.StatusBadRequest, errors.Wrap(err, "Malformed integration ID.")
 	}
 
-	return integrationID, http.StatusOK, nil
+	integrationObject, err := h.IntegrationRepo.Get(r.Context(), integrationID, h.Database)
+	if err != nil {
+		return nil, http.StatusNotFound, errors.Wrap(err, "Failed to retrieve integration object.")
+	}
+
+	return &listOperatorsForIntegrationArgs{
+		AqContext:         aqContext,
+		integrationObject: integrationObject,
+	}, http.StatusOK, nil
 }
 
 func (h *ListOperatorsForIntegrationHandler) Perform(ctx context.Context, interfaceArgs interface{}) (interface{}, int, error) {
-	integrationID := interfaceArgs.(uuid.UUID)
+	args := interfaceArgs.(*listOperatorsForIntegrationArgs)
+
+	integrationID := args.integrationObject.ID
+
+	if args.integrationObject.Service == shared.AWS {
+		// If the requested integration is a cloud integration, substitute the cloud integration ID
+		// with the ID of the dynamic k8s integration.
+		k8sIntegration, err := h.IntegrationRepo.GetByNameAndUser(
+			ctx,
+			fmt.Sprintf("%s:%s", args.integrationObject.Name, dynamic.K8sClusterNameSuffix),
+			uuid.Nil,
+			args.OrgID,
+			h.Database,
+		)
+		if err != nil {
+			return nil, http.StatusInternalServerError, errors.Wrap(err, "Failed to retrieve the implicitly created k8s integration.")
```

Comment:
I feel 'implicit' may be a little confusing to user. Could we say 'Aqueduct created k8s integration' instead?